resources/locale/en.yml

```
@@ -63,6 +65,14 @@ fof-gamification:
         number_title: How many rank badges should be shown?
         help:
           color: '#ffffff'
-          help: "Input the required number of upvotes, the name of the rank, and the hex color of the rank"
+          help: Input the required number of upvotes, the name of the rank, the hex color of the rank and eventually the Flarum group to associate with the user who reach the rank
```

Maybe this could be clearer?

```suggestion
          help: Input the required upvotes, rank name, rank color (hex) and the group to add the user to after achieving the rank
```

---

Yeah, but don't remove eventually, since adding the group isn't mandatory

---

Ah, in that case, maybe optionally would be a better choice?

---

Yes, I'll commit this